### PR TITLE
Add quote for change items/reactivate order

### PR DESCRIPTION
**Pull request by lexerom**
Original https://github.com/Rebilly/api-definitions/pull/1727

### CODE REVIEW

File: openapi/components/schemas/QuoteChangeOrder.yaml [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```
Comment (noon-dawg) [inline openapi/components/schemas/QuoteChangeOrder.yaml:419]: nitpick: [line breaks](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#line-breaks)
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```

File: openapi/components/schemas/QuoteCreateOrder.yaml [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+              type:
+                - 'number'
+                - 'null'
+              format: double
+  paymentTerms:
+    description: Payment terms for the customer which are displayed on the quote.
+    type: string
+  expirationTime:
+    description: |-
+      Date and time when the quote expires. The default expiration time is one month from the time the quote is issued.
```
Comment (noon-dawg) [inline openapi/components/schemas/QuoteCreateOrder.yaml:370]: nitpick:
```suggestion
      Date and time when the quote expires. 
      The default expiration time is one month from the time the quote is issued.
```

File: openapi/components/schemas/QuoteCreateOrder.yaml [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
+    type: string
```
Comment (noon-dawg) [inline openapi/components/schemas/QuoteCreateOrder.yaml:406]: nitpick:
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
    type: string
```

File: openapi/components/schemas/StorefrontQuoteReactivateOrder.yaml [hunk 4]
```diff
@@ -0,0 +1,471 @@
...
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```
Comment (noon-dawg) [inline openapi/components/schemas/StorefrontQuoteReactivateOrder.yaml:419]: nitpick:
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```

File: openapi/components/schemas/QuoteReactivateOrder.yaml [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+  - orderId
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
```
Comment (noon-dawg) [inline openapi/components/schemas/QuoteReactivateOrder.yaml:17]: nitpick:
```suggestion
    description: |-
    Specifies the type of the quote. 
    A quote can be a subscription or a one-time purchase.
```

File: openapi/components/schemas/QuoteReactivateOrder.yaml [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```
Comment (noon-dawg) [inline openapi/components/schemas/QuoteReactivateOrder.yaml:425]: nitpick:
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```

File: openapi/components/schemas/StorefrontQuoteCreateOrder.yaml [hunk 7]
```diff
@@ -0,0 +1,451 @@
...
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```
Comment (noon-dawg) [inline openapi/components/schemas/StorefrontQuoteCreateOrder.yaml]:
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```

File: openapi/components/schemas/StorefrontQuoteChangeOrder.yaml [hunk 8]
```diff
@@ -0,0 +1,465 @@
+type: object
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
```
Comment (noon-dawg) [inline openapi/components/schemas/StorefrontQuoteChangeOrder.yaml]:
```suggestion
    description: |-
    Specifies the type of the quote. 
    A quote can be a subscription or a one-time purchase.
```

File: openapi/components/schemas/StorefrontQuoteChangeOrder.yaml [hunk 9]
```diff
@@ -0,0 +1,465 @@
...
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```
Comment (noon-dawg) [inline openapi/components/schemas/StorefrontQuoteChangeOrder.yaml]:
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```

**Review by lucasxciv — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Pull request merged**